packages/gatsby-source-filesystem/src/create-remote-file-node.js

```diff
@@ -184,6 +184,7 @@ async function processRemoteNode({
   cache,
   createNode,
   auth = {},
+  headers = {},
```

Comment:
should we call it `customHttpHeaders`, `httpHeaders` or `requestHeaders`

Reply:
why not should I changed it to httpHeaders, because I think it is the most appropriate. what do you think?

Reply:
Sorry for late reply - I think `httpHeaders` seems good!

Reply:
@pieh np! I renamed it as you suggested :)